### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by teolemon**
Enjoy!

### CODE REVIEW

**Comment by stephanegigandet** [comment]
Can you resolve the conflict? The web editor doesn't seem to work. We should keep the current nutriscore.html for French.

File: lang/fr/texts/open-food-hunt.html [hunk 1]
```diff
@@ -81,9 +80,9 @@ <h3>Concours Open Food Hunt</h3>
 
 <h3>Devenez un explorateur Open Food!</h3>
 
-<p>The easiest way to start to add products it to install our <a href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=en">Android</a>,
- <a href="https://itunes.apple.com/app/open-food-facts/id588797948">iPhone</a>
- or <a href="https://www.microsoft.com/p/openfoodfacts/9nblggh0dkqr">Windows Phone</a> apps. Vous pouvez également prendre des photos de produits avec votre appareil photo et les mettre en ligne sur le site Web Open Food Facts.</p>
+<p>Le moyen le plus simple de commencer à ajouter des produits est d’installer notre application <a href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=en">Android</a>,
```
Comment (teolemon) [inline lang/fr/texts/open-food-hunt.html]:
```suggestion
<p>Le moyen le plus simple de commencer à ajouter des produits est d’installer notre application <a href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=fr">Android</a>,
```

File: lang/de/texts/index.en.html [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
+
+<p>… und für alle anderen Anwendungen, an die wir nicht gedacht haben! Indem Sie diese Daten kostenlos und für alle Zwecke öffentlich zugänglich machen, sind Sie nicht durch unsere Vorstellungskraft begrenzt, und Menschen aus der ganzen Welt werden Tausende von Nutzungsideen finden, an die wir nie gedacht hätten.</p>
+
+<p>Haben Sie Ideen? Teilen Sie diese im <a href="https://openfoodfacts.uservoice.com/">Ideen-Forum</a>!</p>
+
+<h2>Kürzlich aktualisierte Produkte:</h2>
+
+<p>➜ <a href="/state/to-be-completed">Produkte, die hinzugefügt wurden</a> (insbesondere Produkte, die mit den iPhone, Android und Windows Phone Apps hinzugefügt werden)</p>
+
+[[Abfrage:{"sort_by":"last_modified_t_complete_first"}]]
```
Comment (teolemon) [inline lang/de/texts/index.en.html]:
```suggestion
[[query:{"sort_by":"last_modified_t_complete_first"}]]
```

File: lang/ar/texts/index.en.html [hunk 3]
```diff
@@ -1,40 +1,40 @@
...
+<p>... ولجميع الاستخدامات الأخرى التي لم نفكر فيها! By making this data publicly available for free and for all uses, you are not limited by our imagination,
+and people from all around the world will find thousand of usage ideas that we would never have thought about.</p>
+
+<p>عندك أفكار؟ شاركها على منتدى <a href="https://openfoodfacts.uservoice.com/"> فكرة </a>!</p>
+
+<h2>Recently updated products:</h2>
+
+<p>&rarr; <a href="/state/to-be-completed"> يتم إضافة المنتجات </a> (بعض المنتجات تمت إضافتها بتطبيقات iPhone و Android و Windows Phone)</p>
+
+[[الاستعلام: { "sort_by": "last_modified_t_complete_first"}]]
```
Comment (teolemon) [inline lang/ar/texts/index.en.html]:
```suggestion
[[query:{"sort_by":"last_modified_t_complete_first"}]]
```

**Pull request closed without merging**